components/lib/crash/src/main/java/mozilla/components/lib/crash/service/GleanCrashReporterService.kt

```diff
@@ -34,6 +34,7 @@ class GleanCrashReporterService(
         // These keys correspond to the labels found for crashCount metric in metrics.yaml as well
         // as the persisted crashes in the crash count file (see above comment)
         const val UNCAUGHT_EXCEPTION_KEY = "uncaught_exception"
+        const val CAUGHT_EXCEPTION_KEY = "caught_exception"
```

Comment:
Please also update the label [here](https://github.com/mozilla-mobile/android-components/blob/6c6dc3ce553b1032c5d3e183fb61201e4be39ed2/components/lib/crash/metrics.yaml#L19) in the metrics.yaml to match.  You may need to do a clean and then build for the changes to the metrics.yaml to be picked up by the Glean_Parser.  I don't think that there are any pipeline changes to make as a result of this, but I'm looking into that just to be sure.

Reply:
Nope, looks like the pipeline treats the labels as arbitrary string keys so nothing to change beyond this.

Reply:
Thanks for catching this.